.mocharc.yml

```
@@ -0,0 +1,3 @@
+require:
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)